src/components/Footer/index.js

```diff
@@ -34,23 +57,22 @@ const Footer = () => (
     <Disclaimer>
       <Row>
         <Column span={12}>
-          <Heading as="h2">Lukt het niet om een melding te doen?</Heading>
-          <Paragraph>&nbsp;</Paragraph>
+          <StyledHeading as="h4">Lukt het niet om een melding te doen?</StyledHeading>
```

Comment:
The heading should still be an `h2`; both semantically and visually. `as` Prop should be replaced by `$as`.

Reply:
gedaan